lib/proportions.js

```diff
@@ -76,6 +76,11 @@ define(['d3-interpolate', 'snabbdom', 'filters/genericnode', 'helper'],
 
           return V.h('tr', [th, td]);
         });
+
+        if (items.length === 0) {
+          return null;
```

Comment:
why do you add this return - to get an empty table?
This could cause errors in V.patch function.